gen/installer/bash.py

```diff
@@ -638,12 +638,15 @@ def fill_template(base_name, format_args):
 def make_dcos_launch():
     # NOTE: this needs to be kept in sync with build_dcos_launch.sh
     work_dir = py.path.local.mkdtemp()
+    git_src = 'file://{}'.format(os.getcwd())
+    if os.path.exists('ext/upstream'):
+        git_src = os.path.join(git_src, 'ext/upstream')
+    # clone so the in-place build doesn't affect this tree
+    subprocess.check_call(['git', 'clone', '--progress', git_src, str(work_dir)])
```

Comment:
I do not understand the value of using fetch_git. That function either checks out a mirror clone if the destination doesnt exist or it updates an already existing git repo to have an updated origin.

What I need here is a copy of the working tree so that the spec file can grab test_util/launch.py and any other data (uses local path, not package_data association). The reason for this is when we want to do more advanced things (like running an installer against a VPC) we will need the utilities that pull in these files.

So, we can leave this out entirely for now and go back to just copying the test_util/launch.py file to our working dir, but if/when we do need to have a local copy of the tree, why would I use a function that requires me to make extra function calls? 
1, need to make a new path string that is not a dir yet
2. fetch_git
3. git checkout whatever current HEAD is inherited from cloning the local tree
4. go to objects dir to build
VS
1. subprocess git clone (not a mirror) or recursive copy local tree
2. build in resultant dir